docs/references.md

```diff
@@ -38,3 +38,13 @@
 
 - [Ralf Gommers - Python packaging in 2021 - pain points and bright spots (2021)](https://labs.quansight.org/blog/2021/01/python-packaging-brainstorm/)
 
+
+## General packaging core concepts
+
+- [Python Packaging Authority - Python Packaging User Guide (2010)](https://packaging.python.org/)
+- [Naomi Ceder, for Manning Books - The Quick Python Book, 3rd edition, Chapter 18 (2018)](https://www.manning.com/books/the-quick-python-book-third-edition)
```

Comment:
The "for Manning books" here and further down looks a bit out of place, I'd prefer simply "author - title"